paasta_tools/api/views/instance.py

```diff
@@ -845,3 +845,64 @@ def get_deployment_version(
 ) -> Optional[str]:
     key = ".".join((cluster, instance))
     return actual_deployments[key][:8] if key in actual_deployments else None
+
+
+@view_config(
+    route_name="service.instance.mesh_status", request_method="GET", renderer="json",
+)
+def instance_mesh(request):
+    service = request.swagger_data.get("service")
+    instance = request.swagger_data.get("instance")
+    verbose = request.swagger_data.get("verbose") or 0
```

Comment:
should we use `request.swagger_data.get("verbose", 0)` here?

Reply:
added default values to api specs